tests/install_vm.py

```diff
@@ -198,6 +208,9 @@ def main():
         print(command)
     else:
         os.system(command)
+        if data.console:
+            os.system("unbuffer virsh console {0}".format(data.domain))
+            os.system("virsh start {0}".format(data.domain))
```

Comment:
Do we want to start the machine after installation?

Reply:
I think yes, we want to have the same behavior as without `--console` option.

Reply:
Okay, I agree with that.